### PR TITLE
[MRG] Fix to docstring

### PR DESCRIPTION
**Pull request by jaeilepp**
Continuation for https://github.com/mne-tools/mne-python/pull/2824.

### CODE REVIEW

File: mne/evoked.py [hunk 1]
```diff
@@ -374,9 +374,12 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
             Scale plot with channel (SI) unit.
         show : bool
             Call pyplot.show() at the end or not.
-        ylim : dict
-            ylim for plots. e.g. ylim = dict(eeg=[-200e-6, 200e-6])
-            Valid keys are eeg, mag, grad
+        ylim : dict | None
+            ylim for plots (after scaling has been applied). The value
+            determines the upper and lower subplot limits. e.g.
+            ylim = dict(eeg=[-20, 20]) Valid keys are eeg, mag, grad. If None,
```
Comment (jona-sassenhagen) [inline mne/evoked.py]: Period after the bracket

**Comment by jona-sassenhagen** [comment]
Sorry for missing those the first time around!

**Comment by jasmainak** [comment]
LGTM. +1 for merge.

**Comment by jaeilepp** [comment]
Merged.

**Pull request merged**